exporter.go

```diff
@@ -389,8 +389,8 @@ samples:
 			for _, component := range components[2:] {
 				switch component[0] {
 				case '@':
-					if statType != "c" && statType != "ms" {
-						log.Debugln("Illegal sampling factor for non-counter metric on line", line)
+					if statType == "g" {
+						log.Debugln("Illegal sampling factor for gauge metric type on line", line)
```

Comment:
I'm thinking about the semantics of a sampled gauge … I suppose it would make sense to set it to whatever the last submitted value is, no matter the sampling factor?